src/components/autocompleter/autocompleter.md

```diff
@@ -0,0 +1,17 @@
+The `Autocompleter` component is a wrapper for Select component with the additional functionality searching for a specific models from an API endpoint.
+
+`apiEndpoint`, should be the remainder of the apiEndpoint (excluding the base api), eg. `/workspaces`.
+`value` and `models` shape is expected to be an object(s) with an `id` key, and a display attribute that is one of the following: `title`, `name`, `full_name`, `currency`;
+
+For a11z, and additional prop info, see the `Select` component.
+### Default examples
+```js
+import Autocompleter from './autocompleter.jsx';
+<Autocompleter apiEndpoint={'/models'} />
```

Comment:
It might be good to show a real-ish example here like `<Autocompleter apiEndpoint="workspaces" />`. I think models is pretty clear but using a model that Mavenlink actually has could cement the idea.

Reply:
I kiiiiinda like the abstract example. My intent was to show that it works for anything. Is that more clear with a specific example 🤷‍♂️

Reply:
Works for me! I think it's nit-picky feedback anyway